Liberty30/foodee-example-client
# Changes after rebase

Purpose
---------------
changes after rebase

Solution
---------------
- Where the rebase against the upstream main branch of EC had code that slipped into Foodee, change code to be correct.
- bring in handle and name where we can instead of mocking

Change summary
---------------
* refer to list above


Steps to Verify
----------------
1. check UI upon login, create a new post from EC, and running the populate script

Additional details / screenshot
----------------
- Any supplemental pictures or material
